src/microbe_stage/Patch.cs

```diff
@@ -39,6 +40,29 @@ public Patch(LocalizedString name, int id, Biome biomeTemplate, PatchSnapshot cu
         this.currentSnapshot = currentSnapshot;
     }
 
+    /// <summary>
+    ///   This contains predefined biome types
+    /// </summary>
+    /// <remarks>
+    ///   <para>
+    ///     Explicitly stated values are used as patch ids in classic map
+    ///   </para>
+    /// </remarks>
+    public enum BiomeTypes
```

Comment:
This feels a bit unorderly to be in the Patch class, might need to be in its own file. Also this can probably be used not just as patch IDs in the classic map so the explicit values can be irrelevant. Generalizing this might be a good approach.

Reply:
> Also this can probably be used not just as patch IDs

This already isn't and has been used widely in PatchMapGenenrator.cs
The explicit values also serves a purpose, that in patch generation logic biome types are generated using random.Next() and by using an int circulation variable.

Reply:
Btw I just realized shoudn't this work without explicit values? Enums start from 0 already.

Reply:
It should, but I doubt if one day someone will sort this enum alphabetically then break the whole generation.

Reply:
Documentations are there for a reason.

Reply:
I'm wondering if I can move this to Biome class

Reply:
Biome class is a biome type by itself and this enum should be outside of it in terms of scope.